### ISSUE TITLE
Update text input controls' placeholder color to meet contrast requirements

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#897]
**Describe the bug**

The TextControlPlaceholderForegroundFocused resource uses a color that doesn't meet the 4.5:1 contrast ratio against a white background.

**Steps to reproduce the bug**

1. Go to Xaml Controls Gallery's TextBox or PasswordBox sample page in either light or dark theme
2. Find the instance of the control with placeholder text and put focus on that control
3. Observe that the placeholder text shows as light grey on white background

**Expected behavior**

Placeholder text's foreground color should be #99000000. Suggested name for the new color resource that needs to be added as part of this fix: SystemChromeBlackMediumMediumColor

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/59734824-cdff5580-9207-11e9-9d12-08399c83475a.png)
![image](https://user-images.githubusercontent.com/13142665/59734842-e1aabc00-9207-11e9-815a-bf9e544ed6d2.png)

**Additional context**

Issue originally tracked by an [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/19027035?src=WorkItemMention&src-action=artifact_link).

**Comment by chrisglein** [comment]
Either needs WinUI 3 or needs TextBox/PasswordBox templates to be moved into WinUI. Adding needs-winui-3 label.

**Comment by shaheedmalik** [comment]
They both need to be adjusted.

Black themes shouldn't have a white background for the text box.

The place holder text should be darker for white themes.

**Comment by mdtauk** [comment]
The contrast level needs to be met sure, but it still needs to look sufficiently different to the actually entered text.

@shaheedmalik The Dark Theme doesn't **need** to lose the dark text on white, but there is an argument for changing it.  Aesthetically and for fitting in with the other controls.

iOS 13 does it
![image](https://user-images.githubusercontent.com/7389110/59815138-967bd080-930f-11e9-9c17-200d748b25b0.png)

Material Design does it
![image](https://user-images.githubusercontent.com/7389110/59815775-8bc23b00-9311-11e9-854c-05157852a63d.png)

Outlook UWP Dark Theme does it
![image](https://user-images.githubusercontent.com/7389110/59815831-c0ce8d80-9311-11e9-92df-b52f2edf9efd.png)

**Comment by dpierangeli** [comment]
> Black themes shouldn't have a white background for the text box.

+1

It's true that the white 'focused' state is great to show that the control is the current active control, but it can be more nuanced.

**Comment by chingucoding** [comment]
Given that the Texbox template, PasswordBox and Autosuggestbox templates are in the code base now, should we revisit this issue and update the placeholder colors?

**Comment by shaheedmalik** [comment]
Mail - Dark Mode.
![image](https://user-images.githubusercontent.com/51347950/86650994-2a6b1980-bfb1-11ea-90a0-7bc44d01ef0b.png)
Mail - Dark Mode - Selected
![image](https://user-images.githubusercontent.com/51347950/86651176-4f5f8c80-bfb1-11ea-89bd-acddc1366d9c.png)

**Comment by YuliKl** [comment]
This is updated with new controls styles.
